docs/testing/cart-checkout/general-flow.md

```diff
@@ -0,0 +1,20 @@
+[![Create Todo list](https://git-todo.netlify.app/github-button.png)](https://git-todo.netlify.app/create)
+
+# General Flows
+
+## Setup
+
+This is a general flow of the main functionality of the blocks without going too much
+into details, you're expected to set your store as is, with your current flow, products,
+taxes (if you have any), shipping and so on.
+
+## What to test
+
+- [ ] Create pages with Cart and Checkout blocks.
+- [ ] Set the continue to checkout link your Cart settings to the Checkout page you created.
```

Comment:
Instead of that (or in addition), should we suggest them to select the new pages in _WooCommerce_ > _Settings_ > _Advanced_? Or even to replace the shortcodes for the blocks in the old pages?

Reply:
I was thinking of that but I wanted to simulate a normal case in which most people won’t change the default page, I will add it below.

Reply:
added in f8093eda1